src/styles/blocks/price.scss

```diff
@@ -0,0 +1,24 @@
+.price {
+  display: flex;
+}
+
+.text1 {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 18px;
+  margin-top: 24px;
+  margin-left: 17px;
+  color: #616070;
+}
+
+.text2 {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 18px;
+  margin-top: 24px;
+  margin-left: 90px;
```

Comment:
The use of fixed margins to position elements can lead to issues with responsiveness. If this layout requires alignment with other elements, consider using flexbox properties or CSS Grid to achieve the desired layout, which will be more responsive and maintainable.